python/nimble/ssm.py

```diff
@@ -0,0 +1,4 @@
+from cdktf_cdktf_provider_aws import ssm
+
+def create_new_string_parameter(scope, id, name, value):
```

Comment:
do we need this module @byhbt

Reply:
No, removed in https://github.com/nimblehq/terraform-cloud-development-kit/commit/0e6577b091e1d7485f5b9263a0dd87422ba029b3